Studio-Lovelies/StudioLoveliesWebsite
# Adds smoke tests

## Summary
Wanted to add automated version updates, but realized we don't have any way to make sure the website still works after we upgrade packages.  
We currently do deploy it to `preview.studio-lovelies.com` for each PR, but it might just not respond or return a 404; GitHub wouldn't care and declare the deployment completed successfully.

This PR adds test that ensure that the home page `/` works and `/404` actually returns a 404.  
Consider this a smoke test for the web server.

Also updates the README to contain everything previously only established via chat.

## Testing instructions
1. Make sure what the README says actually works
2. Check that `preview.studio-lovelies.com` still looks fine

## Additional information

- `[ ]` Changes UI
- `[ ]` Has associated resource:
